Source/AcceleratorLattice/LatticeElements/LatticeElementBase.cpp

```diff
@@ -0,0 +1,42 @@
+/* Copyright 2022 David Grote
+ *
+ * This file is part of WarpX.
+ *
+ * License: BSD-3-Clause-LBNL
+ */
+#include "LatticeElementBase.H"
+#include "Utils/WarpXUtil.H"
+#include "Utils/TextMsg.H"
+
+#include <AMReX_ParmParse.H>
```

Comment:
For `amrex::Gpu::copyAsync`:
```suggestion
#include <AMReX_GpuContainers.H>
#include <AMReX_ParmParse.H>
```

Reply:
No, already in `.H` file - all good.